typing/ctype.mli

```diff
@@ -246,6 +246,9 @@ val apply:
 val try_expand_once_opt: Env.t -> type_expr -> type_expr
 val try_expand_safe_opt: Env.t -> type_expr -> type_expr
 
+val path_and_expansion: Env.t -> type_expr -> (Path.t * type_expr) option
+(** Return an option instead of raising, and gives the path of the input type *)
```

Comment:
The comment should say that it looks through `private` types, which I wouldn't otherwise expect.